# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../transcode/hotfix-v4l-detection.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- transcode-1.0.2/acinclude.m4	2005-07-04 08:58:50.000000000 +0200
+++ transcode/acinclude.m4	2006-10-08 00:39:22.000000000 +0200
@@ -92,7 +92,7 @@
 ],    [AC_DEFINE([HAVE_STRUCT_V4L2_BUFFER], 1,
         [define if your videodev2 header has struct v4l2_buffer])
         AC_MSG_RESULT([yes])],
-      [v4l2=no AC_MSG_RESULT([no])])
+      [AC_MSG_RESULT([no])])
   fi
 
   if test x"$v4l" = x"yes" -o x"$v4l2" = x"yes" ; then
